src/python/src/grpc/_adapter/_low_test.py

```diff
@@ -115,6 +115,12 @@ def tearDown(self):
   def _perform_echo_test(self, test_data):
     method = 'test method'
     details = 'test details'
+    server_leading_metadata_key = 'my_server_leading_key'
+    server_leading_metadata_value = 'my_server_leading_value'
```

Comment:
Let's use binary values containing internal null bytes for all three metadata values.